File: _includes/python/code_vec.py
import numpy as np

A = np.array([[1, 2], [3, 4]])
A
# array([[1, 2],
#        [3, 4]])

A.flatten(1)
# array([1, 3, 2, 4])

def vec(A):
    return A.flatten(1)

vec(A)
# array([1, 3, 2, 4])

File: _includes/julia/code_vec.jl
A = [1 2; 3 4]
# 2x2 Array{Int64,2}:
#  1  2
#  3  4

@doc vec
# vec(Array) -> Vector
#
# Vectorize an array using column-major convention.

vec(A)
# 4-element Array{Int64,1}:
#  1
#  3
#  2
#  4
